public/about/about.pug

```diff
@@ -1,30 +1,67 @@
-    p Please keep in mind that the database does not include information from the extended narrative sections of the Dictionary’s entries. For example, in our database, only two travelers appear to have traveled on the Radcliffe Travelling Scholarship (recorded in the ‘Exhibitions and Awards’ filter). In fact there are three more individuals in the Dictionary who were recipients of this scholarship, but this distinction is only mentioned in the extensive narratives of their travels, which we did not include in our parsing--this is also why we have made sure to attach the full text of the Dictionary’s entries to the database’s entries, and to render it searchable throughout by way of “Free word search.”
-    p Two important things to take away:
-    p 
-      strong This application is an explorer. 
-      | It is a tool to search and browse a rich dataset, with results that might well be points of departure for further searches (starting with the word search within the full text of the Dictionary’s entries, while also remembering that the Dictionary itself never was a complete record of what happened--we do not claim to know all the travelers or all there is to know about them!).
-    p 
-      strong The application is still under construction. 
-      | This is a reason for why the access is limited to your group (along with copyright issues that will be sorted out as the final design takes shape: please do not share your login information with anyone else). This also means that mistakes are likely to be there (hopefully only a few!). These might be spelling mistakes or imprecise data to be corrected (please let us know if you see any). It might also be that in coming weeks better ways of doing certain things will become apparent as we engage with this tool in research. Improvements and changes will occur—and they will be communicated clearly.
-    
-  p.text-serif [1] Abbé Gabriel-Francois Coyer, 
-    em Voyage d’Italie et de Hollande
-    | (Paris, 1775), vol. I, p. 4.
+  h1 Terms of Use
+
+  h2.text-serif(style="font-size: 28px; text-align: center; color: #d6bc73;") Copyright
```

Comment:
I think it looks better to have the headings left-aligned instead of center-aligned.

Center-aligned:

![image](https://user-images.githubusercontent.com/1689183/66275964-e9d44900-e842-11e9-9daa-397193468311.png)

Left-aligned: 

![image](https://user-images.githubusercontent.com/1689183/66275944-c6a99980-e842-11e9-8887-c503c4a5f69e.png)